stdlib/public/core/OutputStream.swift

```diff
@@ -415,7 +414,7 @@ internal func _print_unlocked<T, TargetStream : TextOutputStream>(
 ///
 /// This function is forbidden from being inlined because when building the
 /// standard library inlining makes us drop the special semantics.
-@inline(never) @_effects(readonly)
```

Comment:
Is this no longer relevant?

Reply:
Yeah. I'm going to do a pass where I remove all `@inline(never)` calls that aren't also explicitly `@inlinable` in the next one.